### ISSUE TITLE
Accessibility - Individual Goods Screen - Missing Heading

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#43]
The 'Individual goods' screen is missing headings for each section so TalkBack user can tell where they are and use them to navigate the screen.

**Comment by aquilent-dgemoets** [comment]
See discussion for #20.

**Comment by jfc3-dol** [comment]
I reviewed the ILAB 'Sweat and Toil' application on OPA's 'Galaxy Tab S' and an ILAB employees 'LG Tribute 2' and both did not read any headings. 

From my understanding two of the 'More information' screens (About the App and Methodology) where HTML pages and had the headings in them. 

I thought the OSHA 'Heat Index' application had similar screens and the headings with their proper level were read for HTML screens before. The other Android screens that had what equated to headings were read too.

**Comment by aquilent-dgemoets** [comment]
I checked the repository for the (unpublished) Android version of the OSHA 'Heat Index' application and didn't see any added accessibility code/configuration.  Could you confirm that the behavior you are remembering was for the Android version of the app?

**Comment by jfc3-dol** [comment]
Thanks, for correcting this issue. I'm closing this issue.
